index.html

```diff
@@ -538,7 +538,7 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
 
 			</div>
 			<div class="align-bottom text-center">
-				<a class="btn btn-white mb32" href="#">Connect</a>
+				<a class="btn btn-white mb32" href="2018.fossasia.org"></a>
```

Comment:
There is nothing between anchor tags. Why you've removed `Connect` ? Just replace `href="#"` to `href="2018.fossasia.org"`.